examples/compose-your-graphql-router/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "compose-your-graphql-router"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow = "1.0.54"
+apollo-router = { path = "../../apollo-router" }
+apollo-router-core = { path = "../../apollo-router-core" }
+futures = "0.3.21"
+tokio = { version = "1.16.1", features = ["full"] }
+tracing = "0.1.31"
+tokio-util = { version = "0.7.0", features = ["net", "codec"] }
+tracing-subscriber = { version = "0.3.9", features = ["json", "env-filter"] }
+tower = "0.4.12"
+serde_json = "1.0.79"
+serde = "1.0.136"
+serde_json_bytes = "0.2.0"
+apollo-parser = "0.2.3"
+axum = "0.4.6"
```

Comment:
Should we be checking to make sure that the various dependencies have the same features/version as we use in the apollo-router crates?

Reply:
I think many of them can / should be reexported to avoid user hassle tbh, but yeah I'll check it out, thanks!

Reply:
fixed in def7ea6a